udm-le.env

```diff
@@ -144,3 +144,6 @@ UNIFIOS_CERT_PATH="/data/unifi-core/config"
 UNIFIOS_KEYSTORE_PATH="/usr/lib/unifi/data"
 UNIFIOS_KEYSTORE_CERT_ALIAS="unifi"
 UNIFIOS_KEYSTORE_PASSWORD="aircontrolenterprise"
+
+# Appears to be the path for nginx on UCK-G2-PLUS
+                                                              |  UNIFIEUS_CERT_PATH="/data/eus_certificates"
```

Comment:
@Jorel-Master-Of-Scheduling @kchristensen heads up, there looks to be an errant `|` introduced here.